src/components/Inputs/ColorPicker.tsx

```diff
@@ -31,21 +32,34 @@ interface ColorPickerProps {
 }
 
 const ColorPicker = ({ value, onChange }: ColorPickerProps) => {
+  const { t } = useTranslation('App')
   const color = value?.toString() || getRandomLabelColor()
   const [isPopupOpen, setIsPopupOpen] = useState(false)
   const ref = useDetectClickOutside({ onTriggered: () => setIsPopupOpen(false) })
 
+  const handlePopupOpen = () => setIsPopupOpen(!isPopupOpen)
+  const onChangeComplete = (newColor: { hex: string }) => {
+    onChange(newColor.hex)
+    handlePopupOpen()
+  }
+
   return (
     <ColorPickerContainer ref={ref}>
-      <Input onClick={() => setIsPopupOpen(!isPopupOpen)}>
+      <Input
+        role="button"
+        tabIndex={0}
+        aria-label={t`Pick a color`}
+        onClick={handlePopupOpen}
+        onKeyPress={handlePopupOpen}
```

Comment:
Maybe we could have a component like `ClickableDiv` or sth like that where by default it has all these properties set. I see them being repeated quite often in this PR ( ex:  `AddressOption`, `ExpandableSection Title`, `ActionButtonContainer`, etc). @LeeAlephium WDYT?

Reply:
Explained offline: it's pretty situational but there are probably ways to make it DRYer, we can iterate